mannaggia.sh

```diff
@@ -101,7 +109,20 @@ while [ "$nds" != 0 ]
 			# attenzione: se non siete root o sudoers dovete togliere dalla riga successiva "sudo" e "-n"
 			echo "$MANNAGGIA" | sudo wall -n
 		fi
+	#mannaggia via notifica desktop
+	elif [ "$notify" = 1 ]; then
+		if  ( ! type "notify-send" >/dev/null  2>&1 ); then
+			echo "Ti serve notify-send"
+			exit 1
 		else
+        		notify-send  "$MANNAGGIA" 2>/dev/null
+		fi
+	elif [ "$help" = 1 ]; then
```

Comment:
Da un punto di vista puramente logico, se l'utente chiede l'help ogni azione dovrebbe essere soppressa, quindi il check dovrebbe essere in cima